source/2019-10-18-the-ember-times-issue-120.md

```diff
@@ -34,13 +34,15 @@ READMORE
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [A New EmberMap Series: What's New in Ember 📚](https://embermap.com/topics/what-s-new-in-ember/)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+This year, Ember released many new features that will all be a part of Ember Octane. While the [release notes](https://blog.emberjs.com/tags/releases.html) do a great job of summarizing the features, you may be still wondering about what they are and why they matter to you as a developer or an addon contributor.
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+To help you understand better, EmberMap started a series called [What's New in Ember](https://embermap.com/topics/what-s-new-in-ember/). Each episode will cover 1 new feature, starting with Ember 3.8.
+
+In the first episode, [Sam Selikoff (@samselikoff)](https://github.com/samselikoff) shows how you can write a **modifier**. In particular, you will learn how to use Ember's **modifier manager** (this provides low-level, public APIs for customization) or dedicated addons (for high-level abstraction).
+
```

Comment:
I would expect the `dedicated addons` to also be bold. Now it kinda got me confused. Might also be because you used `or` instead of what probably could be `and`

Reply:
I see. I had used `or` because I had initially tried to write "use either A or B". I think `and` works just as well, if not better.